plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -7,17 +7,17 @@ const {
 	CUSTOMER_USER_EMAIL,
 	CUSTOMER_FIRST_NAME,
 	CUSTOMER_LAST_NAME,
+	LANGUAGE,
```

Comment:
While reviewing PR's for 7.8, I encountered one PR where a contributor sets up their site and store languages differently. 
WooCommerce Dashboard inside WP Admin is set to `en` language to be understood by  all of their admins.
While their store is set in `es` because that's the language of their customers.

Currently `LANGUAGE` and `langCode` assumes that our e2e tests will use only one language  for the admin- and customer-facing pages. And I think there's a handful of `merchant` tests that navigate to the shop to verify assertions. I think for now we could  leave this single-language support as it is, and worry about how to handle different languages between the admin pages and shop pages at a later time. Just wanted to bring up that there's indeed a use case where their languages are different, as something to ponder on.

Reply:
This is a great consideration Del. Initially, the 'prototype' is to run the tests against EN, ES and AR and I had only considered single-language support. The more coverage we can provide, the better, so this is certainly one to ponder going forward. Thanks!